### PR TITLE
Gridworld ai console

### PR DESCRIPTION
**Pull request by ricvolpe**
Modifies training pipeline to allow for training with gridworld data

### CODE REVIEW

File: extractors/configs/hptuning_config_tf_baseline.yaml [hunk 1]
```diff
@@ -30,29 +30,29 @@ trainingInput:
         maxValue: 0.5
         scaleType: UNIT_LOG_SCALE
 
-      - parameterName: gin_bindings=cnn_from_obs.cnn_first_size
-        type: INTEGER
-        minValue: 2
-        maxValue: 64
-        scaleType: UNIT_LINEAR_SCALE
+#       - parameterName: gin_bindings=cnn_from_obs.cnn_first_size
```
Comment (wichersn) [inline extractors/configs/hptuning_config_tf_baseline.yaml:33]: I  think there should be different config files for each of the datasets.

File: extractors/data_processing.py [hunk 2]
```diff
@@ -29,13 +29,8 @@ def transform_to_x_y(raw_data, env, shuffle_data = True):
...
     xs = np.array(xs)
     ys = np.array(ys).astype(int)
-    
-    if shuffle_data:
-        randomize = np.arange(len(xs))
-        np.random.shuffle(randomize)
-        xs = xs[randomize]
-        ys = ys[randomize]
-        
+    print("xs", xs.shape, "ys", ys.shape)
```
Comment (wichersn) [inline extractors/data_processing.py:32]: Why not shuffling anymore?

File: extractors/tf_extractor.py [hunk 3]
```diff
@@ -89,7 +89,7 @@ def cnn_from_obs(input_shape, cnn_first_size, cnn_last_size, cnn_num_layers, cnn
             stride = 2
         else:
             stride = 1
-        layers.append(tf.keras.layers.Conv2D(layer_size, 3, strides=stride, activation='relu',
+        layers.append(tf.keras.layers.Conv2D(layer_size, 2, strides=stride, activation='relu',
```
Comment (wichersn) [inline extractors/tf_extractor.py:92]: The 2 should be a hyperparam

**Review by wichersn — CHANGES_REQUESTED** [review]

File: extractors/data_processing.py (hunk 2 above)
Comment (ricvolpe) [reply extractors/data_processing.py:32]: Because we already do so in `Extractor.train_single_shuffle` so I removed the duplication
